### PR TITLE
Various documentation improvements

### PR DESCRIPTION
**Pull request by sebhz**
* Improved manpage
    * Generated an HTML manpage from the groff page (using man2html)
      and added it to the repository
    * Referenced the HTML manpage from the README.

### CODE REVIEW

File: doc/man.html [hunk 1]
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```
Comment (ColinPitrat) [inline doc/man.html]: Could we add a target in the makefile to generate it ?

Comment (sebhz) [inline doc/man.html]: Do you want a specific target to generate it, or generate it systematically when compiling the executable ? Just asking to know if we need to implement a check for the existence of man2html or similar converters. Also we would still need to commit the change to the html (if any) manually - but this cannot  really be helped.

Comment (ColinPitrat) [inline doc/man.html]: I was thinking of a specific target (e.g: doc).
It's more to have the command (and eventually options) somewhere to know how to do it next time we (especially I :-)) need it.
We can also imagine adding some check in travis (the continuous integration) to ensure that the HTML is up to date (e.g: `make doc && git status | grep html || fail`) but that's more a nice to have.

Comment (sebhz) [inline doc/man.html]: OK - makefile modified.

**Pull request merged**